app/code/Magento/Sales/Model/AdminOrder/Create.php

```diff
@@ -28,11 +28,6 @@
  */
 class Create extends \Magento\Framework\DataObject implements \Magento\Checkout\Model\Cart\CartInterface
 {
-    /**
-     * Xml default email domain path
-     */
-    const XML_PATH_DEFAULT_EMAIL_DOMAIN = 'customer/create_account/email_domain';
```

Comment:
@dmytro-ch , of course I can leave it as it is. But how to be with config declaration in the `system.xml`?

Thank you!

Reply:
>  Mark it with @deprecated tag instead.

**done**

Thank you, @dmytro-ch

Reply:
@atwixfirster Since this code goes to 2.4.0 there's no sense to remove a field (which is MAJOR change) and leave a corresponding constant (also MAJOR). Please remove it as well and then we go with approval for that.

Reply:
> @atwixfirster Since this code goes to 2.4.0 there's no sense to remove a field (which is MAJOR change) and leave a corresponding constant (also MAJOR). Please remove it as well and then we go with approval for that.

**done**

@dmytro-ch , could you please approve?

Thank you!